cybsi/api/client.py

```diff
@@ -4,7 +4,7 @@
 from .artifact import ArtifactsAPI, ArtifactsAsyncAPI
 from .auth import APIKeyAuth, APIKeysAPI
 from .client_config import DEFAULT_LIMITS, DEFAULT_TIMEOUTS, Limits, Timeouts
-from .custom_lists import CustomListsAPI
+from .custom_list import CustomListAPI, CustomListsAsyncAPI
```

Comment:
А здесь кажется всё ок было с названием. Сам класс во мн.ч
```suggestion
from .custom_list import CustomListsAPI, CustomListsAsyncAPI
```